### PR TITLE
Kyber extension UI avoiding keyboard

### PR DESCRIPTION
**Pull request by flexsurfer**
Updated extension: https://status-im.github.io/pluto/try.html?hash=QmVe4JFe15BUhELhnUTLegcWF72W9LobgrURDgkL6Dp84h

<blockquote><div><strong><a href="https://status-im.github.io/pluto/index.html">Pluto · A grammar for data manipulation</a></strong></div><div>A grammar for data manipulation</div></blockquote>

### CODE REVIEW

File: src/status_im/ui/screens/wallet/settings/views.cljs [hunk 1]
```diff
@@ -58,7 +58,7 @@
 (defview settings-hook []
   (letsubs [{:keys [label view]} [:get-screen-params :wallet-settings-hook]
             {address :address} [:account/account]]
-    [react/view {:style {:flex 1 :background-color colors/white}}
+    [react/keyboard-avoiding-view {:style {:flex 1 :background-color colors/white}}
```
Comment (jeluard) [inline src/status_im/ui/screens/wallet/settings/views.cljs]: This one can't be purely done in the extension, right?

Comment (flexsurfer) [reply src/status_im/ui/screens/wallet/settings/views.cljs]: right, unfortunately

Comment (flexsurfer) [reply src/status_im/ui/screens/wallet/settings/views.cljs]: we can think later, how we can improve this

**Review by jeluard — APPROVED** [review]

**Review by dmitryn — APPROVED** [review]

**Comment by jeluard** [comment]
@flexsurfer Oups looks like you have to rebase!

**Comment by flexsurfer** [comment]
Done

**Comment by Serhy** [comment]
Looks good with Android 6, 7, 8, iOS 12.1.2.

**Pull request merged**